GVFS/GVFS.Installer.Mac/uninstall_vfsforgit.sh

```diff
@@ -3,6 +3,8 @@
 KEXTFILENAME="PrjFSKext.kext"
 VFSFORDIRECTORY="/usr/local/vfsforgit"
 PRJFSKEXTDIRECTORY="/Library/Extensions"
+LAUNCHDAEMONDIRECTORY="/Library/LaunchDaemons"
+PLISTFILENAME="org.vfsforgit.prjfs.PrjFSKextLogDaemon.plist"
```

Comment:
It might be worth giving this a slightly more specific name than `PLISTFILENAME`. It's a bit like calling it `XMLFILENAME` - it tells you the format, not what it's actually for. Is `LOGDAEMONLAUNCHDFILENAME` too long-winded? (More specifically than plist, this is a launchd plist, as described in `man 5 launchd.plist`)